### DISCUSSION TITLE
Issues with gsync/FPS locking while using Awakened PoE Trade

### DISCUSSION THREAD

**Discussion by Traceur10** [SnosMe/awakened-poe-trade#1481] [Q&A]
I upgraded to a GSYNC monitor and the monitor has a built in frames-per-second indicator that overlays on the screen. When I open Path of Exile, the in-game FPS counter and the monitor FPS counter both overlap/sync perfectly, so I know GSYNC is working correctly. But unfortunately, as soon as i open Awakened PoE Trade, the monitor FPS overlay pegs at 240hz and never moves while the in-game FPS counter continues to fluctuate. As soon as I close Awakened PoE Trade, the counters sync again. 

Any ideas on what I can do or things I can try to keep GSYNC synced with PoE instead of Awakened PoE Trade? 

My guess is it's hardware acceleration or something being used in the background of Awakened PoE Trade app. Happy to take any and all suggestions. I can also take photos/videos of any behaviors if it helps. Just let me know.

Thanks,

**Comment by Fugaz1** [comment]
I recently had same problem. Solution was quite unusual it was RTX video scaling causing this because I had firefox with twitch open. I disabled rtx video upscaling and gsync is working without problems.
